.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -29,8 +29,11 @@ React Native version:
 Describe what you expected to happen:
 
 
-[Snack](https://snack.expo.io/), screenshot, code example, or link to a repository:
+Snack, screenshot, code example, or link to a repository:
 
+<!--
```

Comment:
Less is better when it comes to issue templates. How about we expand the existing comment block below instead of adding a new one? For example:

```
<!--
  Please provide a Snack (https://snack.expo.io/), a link to a repository on GitHub, or
  provide a minimal code example that reproduces the problem.
  You may provide a screenshot of the application if you think it is relevant to your bug report. 
  Here are some tips for providing a minimal example: https://stackoverflow.com/help/mcve.	  --	
-->
```

Reply:
Thank you very much for helping me with this!